index.html

```diff
@@ -1,134 +1,368 @@
+        <span id='instructions' class="h3 text-info" data-i18n="step1.desc"></span>
+      </div>
+      <div class='mb-4 mt-3'>
+        <form id='find' class="input-group">
+          <input class="form-control" id="address" type="search" placeholder='' data-i18n="[placeholder]step1.addr" />
+          <div class="input-group-append">
+            <button class="cbtn btn-primary px-3" id='findme'><span data-i18n="step1.button"></span><span class="spinner-grow spinner-grow-sm invisible ml-2" role="status" aria-hidden="true"></span></button>
+          </div>
+        </form>
+        <p id="couldnt-find" class="alert alert-danger mt-2" role="alert" style='display: none;' data-i18n="[html]step1.fail"></p>
+      </div>
+      <div id='findme-map'></div>
+      <a class="btn btn-primary btn-lg text-white w-100 mt-3 text-uppercase disabled" id="continue" href="#details" data-i18n="step1.continue"></a>
+    </div>
+
+    <!-- step 2 -->
+    <div class='mt-3 d-none' id='collect-data-step'>
+      <p data-i18n="[html]step2.desc" class="h3 text-info"></p>
+      <div>
+        <!-- categories -->
+        <div class="row">
+          <div class="form-group col">
+            <label for='category' data-i18n="step2.catlabel"></label>
+            <input id='category' data-placeholder='' class="w-100" data-i18n="[data-placeholder]step2.catplaceholder" />
+          </div>
+          <div class="form-group col-sm-7">
+            <label for='categoryalt' data-i18n="step2.catalt"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-convenience-store"></i></span>
+              </div>
+              <input id='categoryalt' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.cataltplaceholder" />
+            </div>
+          </diV>
+        </div>
+
+        <hr>
+        <p data-i18n="step2.addressalt" class="mt-3"></p>
+        <!-- address 1 -->
+        <div class="form-row">
+          <div class="form-group col-md-5">
+            <label for='addressalt' data-i18n="step2.street"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-round-pushpin"></i></span>
+              </div>
+              <input id='addressalt' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.addressaltplaceholder" />
+            </div>
+          </div>
+          <div class="form-group col-md-2">
+            <label for="housenumber" data-i18n="step2.housenumber"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-keycap-10"></i></span>
+              </div>
+              <input type="number" class="form-control" id="housenumber" data-i18n="[placeholder]step2.housenumberplaceholder">
```

Comment:
input type="text"... considera anche "BIS", "C", "ROSSO/NERO" ecc...ecc...

Reply:
magari chiamalo "hnumberalt"... O simili... così che il browser non provi ad autocompletare il campo :)

Reply:
Vero, non avevo pensato che il type number causa anche un problema di autocorrezione da parte dei browser... Cambio in text. Non sapevo autocompletasse perché io ho sempre disattivato l'autocompletamento sui miei browser. Gli cambio nome allora.